sirepo/package_data/static/js/cloudmc.js

```diff
@@ -544,6 +555,24 @@ SIREPO.app.factory('tallyService', function(appState, cloudmcService, utilities,
         appState.autoSave();
     };
 
+    self.updateThresholds = () => {
+        const s = Object.keys(self.cachedSettings);
+        // if none of the data-specific settings changed, do not update the thresholds
+        if (s.every((k) => self.cachedSettings[k] === appState.models.openmcAnimation[k])) {
+            return;
+        }
+        s.forEach((k) => {
+            self.cachedSettings[k] = appState.models.openmcAnimation[k];
+        });
+        const t = appState.models.openmcAnimation.thresholds;
+        t.global = [self.minField, self.maxField];
+        // since tallies are counts, compare the lower threshold to 0
+        // instead of the global min
+        t.val[0] = Math.max(t.val[0], 0);
+        t.val[1] = Math.min(t.val[1], t.global[1]);
+    };
+
```

Comment:
When you discard changes to the Tokamak Reactor example and run the simulation, it shows an error about the default threshold:
![x](https://github.com/radiasoft/sirepo/assets/6516910/3ffbf104-38cc-4f9b-9e9c-16838b00180e)

It should probably set the lower/upper to the min/max value when initialized. 

Also, if you switch from the t1 tally to the t2 tally, it resets the theshold to the new limits, but if you return to t1, it is showing a constrained result:
![x2](https://github.com/radiasoft/sirepo/assets/6516910/533e7e73-260b-4a3b-bea5-160b69812380)

I think we may want to keep track of the selected threshold per tally, or reset it to the tally min/max when switching between tallies.

Reply:
[8678f98](https://github.com/radiasoft/sirepo/pull/6781/commits/8678f98d491108e1a5c3c4628ef68e23de6c1fc0)

This just moves the lower threshold to 0 if it crosses the upper when changing the data plotted. Tracking every tally/score/aspect combination with its own thresholds seems too complicated to me